gammagl/models/HeCo.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import tensorlayerx as tlx
+import tensorlayerx.nn as nn
+from layers.attention.HeCo_encoder import Mp_encoder
+from layers.attention.HeCo_encoder import Sc_encoder
+r"""HeCo proposed in `"Self-supervised Heterogeneous Graph Neural Network with Co-contrastive Learning" 

```

Comment:
Ditto.

Reply:
sorry for that! I have changed Mp_encoder and Sc_encoder files inside, it has no error on my laptop.

Reply:
You tried the document generation, not just running the code?

Reply:
I might have tried all link relations of the code on my laptop(including data generation etc, while I was running HeCo_trainer.py)and all works well. Could you please give me some hints about the document generation?